vignettes/vignette.Rmd

```diff
@@ -33,6 +33,7 @@ Now that the package is loaded, you will have access to three functions.
 Let's start with the stdizer function!
 
 ```{r}
+library(gapminder)
 # Load gapminder data set
 df <- gapminder::gapminder
```

Comment:
`library(gapminder)` shouldn't be necessary if we have `gapminder::gapminder` call.

Reply:
There is an error in Travis: 

Error: processing vignette 'vignette.Rmd' failed with diagnostics:
there is no package called 'gapminder'